File: output/Palindrome.py


def main():
	s1 = 'sadas'
	i = 0
	while i<int(len(s1)/2):
		if s1[i] != s1[len(s1)-i-1]: 
			print("s1 is not palindrome\n"% (), end = '')
			return 0
		i += 1
	print("s1 is palindrome\n"% (), end = '')
	return 0


if __name__ == '__main__': 
	main()

File: testFiles/Palindrome.c
int main(){
    char s1[] = 'sadas';
    for(int i = 0; i < (int)(strlen(s1) / 2); i++) {
        if(s1[i] != s1[strlen(s1) - i - 1]) {
            printf("s1 is not palindrome\n");
            return 0;
        }
    }
    printf("s1 is palindrome\n");
    return 0;
}